Makefile

```diff
@@ -1,85 +1,41 @@
-DESCRIPTION = CS50 Library for Java
-MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
-OLD_NAME = lib50-java
-NAME = libcs50-java
-VERSION = 2.0.0
-
-BUILD_DIR = build
-DEB_DIR = $(BUILD_DIR)/deb
-DOCS_DIR = docs
-JAR_DIR = $(BUILD_DIR)/usr/java/packages/lib/ext
-PACMAN_DIR = $(BUILD_DIR)/pacman
-RPM_DIR = $(BUILD_DIR)/rpm
-
-.PHONY: all
-all: clean deb pacman rpm
-
-.PHONY: bash
-bash:
-	docker run -i --rm -t -v "$(PWD):/root" cs50/cli
+DESTDIR ?= /usr/local/share/java
+VERSION = 2.0.1
 
 .PHONY: build
 build: clean Makefile src/edu/harvard/CS50.java
-	mkdir -p "$(JAR_DIR)"
-	javac -d "$(BUILD_DIR)" src/edu/harvard/CS50.java
-	jar cvf "$(JAR_DIR)/cs50.jar" -C "$(BUILD_DIR)" .
-	find "$(BUILD_DIR)" -type d -exec chmod 0755 {} +
-	find "$(BUILD_DIR)" -type f -exec chmod 0644 {} +
+	mkdir -p build/usr/share/java
+	javac -d build src/edu/harvard/CS50.java
+	cp src/edu/harvard/CS50.java build/edu/harvard
+	jar cvf build/usr/share/java/cs50.jar -C build .
```

Comment:
Mind reviewing this thread, https://lists.debian.org/debian-java/2014/11/msg00020.html? Seems to confirm your inclination just to update `CLASSPATH`. Though it seems strange to mandate that Java libraries be installed in `/usr/share/java`, but then not even look there by default.